docker/sandbox-bundled/manifests/complete.yaml

```diff
@@ -412,32 +412,15 @@ data:
     \   disableClusterResourceManager: false\n    disableScheduler: false\n    disabled:
     false\n    seedProjects:\n    - flytesnacks\n  dataCatalog:\n    disabled: false\n
     \ propeller:\n    disableWebhook: false\n    disabled: false\n"
-  001-plugins.yaml: |
-    tasks:
-      task-plugins:
-        enabled-plugins:
-          - container
-          - sidecar
-          - K8S-ARRAY
-        default-for-task-types:
-          - container: container
-          - container_array: K8S-ARRAY
-    plugins:
-      logs:
-        kubernetes-enabled: true
-        kubernetes-template-uri: http://localhost:30080/kubernetes-dashboard/#/log/{{.namespace }}/{{ .podName }}/pod?namespace={{ .namespace }}
-        cloudwatch-enabled: false
-        stackdriver-enabled: false
-      k8s:
-        co-pilot:
-          image: "cr.flyte.org/flyteorg/flytecopilot:v0.0.30"
-      k8s-array:
-        logs:
-          config:
-            kubernetes-enabled: true
-            kubernetes-template-uri: http://localhost:30080/kubernetes-dashboard/#/log/{{.namespace }}/{{ .podName }}/pod?namespace={{ .namespace }}
-            cloudwatch-enabled: false
-            stackdriver-enabled: false
+  001-plugins.yaml: "plugins:\n  logs:\n    kubernetes-enabled: true\n    kubernetes-template-uri:
+    http://localhost:30080/kubernetes-dashboard/#/log/{{.namespace }}/{{ .podName
+    }}/pod?namespace={{ .namespace }}\n    cloudwatch-enabled: false\n    stackdriver-enabled:
+    false\n  k8s:\n    co-pilot:\n      image: \"cr.flyte.org/flyteorg/flytecopilot:v0.0.30\"\n
+    \ k8s-array:\n    logs:\n      config:\n        kubernetes-enabled: true\n        kubernetes-template-uri:
+    http://localhost:30080/kubernetes-dashboard/#/log/{{.namespace }}/{{ .podName
+    }}/pod?namespace={{ .namespace }}\n        cloudwatch-enabled: false\n        stackdriver-enabled:
+    false\n  agent-service: \n        defaultGrpcEndpoint: flyte-sandbox-http.flyte.svc.cluster.local:8000\n
+    \       supportedTaskTypes:\n        - bigquery_query_job_task\n"
```

Comment:
why do we need this change?

Reply:
It's auto-generated. I just fixed indentation errors.